### ISSUE TITLE
Problématique d'utilisation et d'opération du composant menu

### ISSUE THREAD

**Issue by GuillaumeDamour** [wet-boew/wet-boew#7123]
Présentation du projet
Le projet du Laboratoire d’Expérimentation et de Promotion de l’Accessibilité du Web a pour but d’aider la Boite à Outils de l’Expérience Web à rendre ses composants plus accessible. Pour plus d’informations aller voir le site : http://labo.raamm.org/a-propos/
Il est important pour nous que l’ensemble des outils d’accessibilité, navigateurs web et systèmes d’exploitation soient testés  sur l’ensemble des plateformes, afin de pouvoir cibler les différentes problématiques de l’accessibilité. C’est pourquoi nous avons recruté un groupe de trente utilisateurs débutants et expérimentés et un groupe de seize experts aux environnements diversifiés.
Nous avons débuté le premier test le 18 juin 2015. Les testeurs étaient invités à se rendre sur une page web spécialement conçue à cet effet : http://labo.raamm.org/test/test-1-le-menu/
Plus de 25 utilisateurs, ainsi qu’une douzaine d’experts ont testé le composant du menu. Certaines personnes l’ayant expérimenté sur plus d’une plateforme, nous avons reçu un total de 46 rapports de test.

Problématique : utilisation et opération
Plusieurs testeurs nous ont indiqué avoir éprouvé de la difficulté à revenir facilement au menu après avoir sélectionné un élément les dirigeant vers le contenu de la page.

Recommandation : ajouter un raccourci clavier pour revenir au menu facilement
Il serait utile d’ajouter une commande permettant de facilement revenir au menu. Nous suggérons : Ctrl + M

**Comment by LaurentGoderre** [comment]
@dfarough @SBleeks @mysticjeff what are your thoughts on this?

**Comment by SBleeks** [comment]
I don’t think that creating a keyboard shortcut command “Ctrl M” is the way to go here.  If the page is coded properly, with headings etc. Etc. , navigation should be easy enough for all levels of users.

**Comment by hsrudnicki** [comment]
S'il vous plaît mettre à jour le statut et le dilemme de ce problem : utilisation et operation? Merci --
Please continue and update the status of this issue as well as a dilemma based on the last comment in regards to understanding the menu? Thank you

**Comment by dfarough** [comment]
I agree with @SBleeks.  It is difficult to get all the players to agree on supporting a particular feature.  Heading and landmark support is far from universal.  Everyone does menus differently and it is hard to imagine a way that something like this could be reliable for all menus.

**Comment by hsrudnicki** [comment]
Based on the above, can be this issue closed as resolved with no fix or if otherwise?  Either way, please update this issue's status, whether this issue can be closed or otherwise, thank you

**Comment by hsrudnicki** [comment]
Can be this issue closed as resolved or otherwise? Please update this issue's status, thx

**Comment by LaurentGoderre** [comment]
@GuillaumeDamour quelle technologies ces utilisateurs utilisaient-ils? Il ne naviguait pas la page avec les entêtes?

**Comment by GuillaumeDamour** [comment]
Nous avons testé le menu dans une page spécialement conçu pour ce composant. http://labo.raamm.org/lab/tests/menu/index.html

Le menu se trouve dans le haut de la page et n'a pas de titre qui le précède, comme on pourrait le trouver dans une page sur le web.

Les technologies les plus utilisés pour le test sont : NVDA avec Firefox, Jaws avec IE et VoiceOVer avec Safari.

**Comment by LaurentGoderre** [comment]
@GuillaumeDamour le menu doit etre teste dans le gabarit, sinon des parties critique pour l'accessibilité manquent et cela peut causer ce genre de probleme. Serait-il possible de re-tester avec une page qui contient tout le gabarit?

**Comment by GuillaumeDamour** [comment]
J'ai fait une vérification et effectivement vous avez mis un H2 dans la balise du menu. Il n'est toutefois pas recommandé de mettre un H2 avant le H1 de la page. La navigation est possible bien que ce ne soit pas recommandé. Je ferme donc cette requête. Merci.
